hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/OzoneClientProducer.java

```diff
@@ -107,7 +108,7 @@ public S3Auth getSignature() {
     } catch (Exception e) {
       // For any other critical errors during object creation throw Internal
       // error.
-      LOG.debug("Error during Client Creation: ", e);
+      LOG.error("Error during Client Creation: ", e);
       throw wrapOS3Exception(S3ErrorTable.newError(INTERNAL_ERROR, null, e));
```

Comment:
I don't think this change is necessary, `S3ErrorTable.newError` also logs internal errors.

https://github.com/apache/ozone/blob/c23aae0d9384a6ad815dc3ad9edeb3cd935c53a5/hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/exception/S3ErrorTable.java#L141-L142

Reply:
Oh, seems it's already being handled. 
Thanks for the remind.